tests/system/providers/google/cloud/dataproc/example_dataproc_batch.py

```diff
@@ -58,7 +61,17 @@
         region=REGION,
         batch=BATCH_CONFIG,
         batch_id=BATCH_ID,
-        timeout=5.0,
+    )
+    # [END how_to_cloud_dataproc_create_batch_operator]
+
+    # [START how_to_cloud_dataproc_create_batch_operator]
```

Comment:
There should be only one marker with the name ``how_to_cloud_dataproc_create_batch_operator``. Either include all two operator examples in single START/END block or use different name (but then remember to also include it in the documentation).

Reply:
I thought if we have 2 examples then there should be doc strings for all of them :)
Thanks!
I have removed duplicated doc strings.